spec/extensions/linear_algebra_functions.md

```diff
@@ -125,7 +125,7 @@ Returns the cross product of 3-element vectors. If `x1` and `x2` are multi-dimen
 (function-linalg-det)=
 ### linalg.det(x, /)
 
-Returns the determinant of a square matrix (or stack of square matrices) `x`.
+Returns the determinant of a square matrix (or a stack of matrices) `x`.
```

Comment:
IMO it's better to not remove the qualifiers from "stack of matrices" (except for the change in `pinv` where it was wrong).

Reply:
The only problem with that is that sometimes it can be a bit overwhelming (think of the case "symmetric positive definite"). Should we still go for qualifiers regardless?

Reply:
```suggestion
Returns the determinant of a square matrix (or a stack of square matrices) `x`.
```